examples/grpc-krotodc/src/main/kotlin/example/armeria/grpc/krotodc/Main.kt

```diff
@@ -0,0 +1,81 @@
+package example.armeria.grpc.krotodc
+
+import com.linecorp.armeria.common.grpc.GrpcSerializationFormats
+import com.linecorp.armeria.server.Server
+import com.linecorp.armeria.server.docs.DocService
+import com.linecorp.armeria.server.docs.DocServiceFilter
+import com.linecorp.armeria.server.grpc.GrpcService
+import example.armeria.grpc.krotodc.Hello.HelloRequest
+import io.grpc.protobuf.services.ProtoReflectionService
+import io.grpc.reflection.v1alpha.ServerReflectionGrpc
+import org.slf4j.LoggerFactory
+
+object Main {
+
+    @JvmStatic
+    fun main(args: Array<String>) {
+        val server = newServer(8080, 8443)
+
+        server.closeOnJvmShutdown()
+
+        server.start().join()
+        server.activePort()?.let {
+            val localAddress = it.localAddress()
+            val isLocalAddress = localAddress.address.isAnyLocalAddress ||
+                localAddress.address.isLoopbackAddress
+            logger.info(
+                "Server has been started. Serving DocService at http://{}:{}/docs",
+                if (isLocalAddress) "127.0.0.1" else localAddress.hostString,
+                localAddress.port
+            )
+        }
+    }
+
+    private val logger = LoggerFactory.getLogger(Main::class.java)
+
+    fun newServer(httpPort: Int, httpsPort: Int, useBlockingTaskExecutor: Boolean = false): Server {
+        val exampleRequest: HelloRequest = HelloRequest.newBuilder().setName("Armeria").build()
+        val grpcService = GrpcService.builder()
+            .addService(HelloServiceImpl())
+            // See https://github.com/grpc/grpc-java/blob/master/documentation/server-reflection-tutorial.md
+            .addService(ProtoReflectionService.newInstance())
+            .supportedSerializationFormats(GrpcSerializationFormats.values())
```

Comment:
Let's remove this line 😉 
```suggestion
```
If you copy this code from another source, please remove it as well. 🙇